### DISCUSSION TITLE
Allow suggesting changes when reviewing a PR by commits

### DISCUSSION THREAD

**Discussion by lwasyl** [community/community#15626] [Pull Requests] [answered]
Right now, when reviewing a PR I can request changes for some lines. That's not possible if I review a PR by opening its commits list and browsing single commit at a time. This is a request to allow suggestions when reviewing by commits — while it looks like it's already possible to add a suggestion manually (by typing ```suggestion), there's no dedicated button for that

**Comment by kreintjes** [comment, marked as answer]
Seems to be a duplicate of https://github.com/github-community/community/discussions/7128. Maybe better to upvote that feature request?
